### ISSUE TITLE
Add support for architectures other than amd64

### ISSUE THREAD

**Issue by djcampello** [canonical/ubuntu-package-buildinfo#5]
### Description

Tool returns buildlog for arm64 but buildinfo for amd64

### What I Did

```
$ python3 ubuntu-package-buildinfo/ubuntu_package_buildinfo/cli.py --series jammy --package-name python3-defaults --package-version 3.10.6-1~22.04 --package-architecture arm64
INFO: 	Found binary package from source package python3-defaults arm64 version 3.10.6-1~22.04 with in jammy.
INFO: 	changes written to python3-defaults_3.10.6-1~22.04_amd64.changes
INFO: 	buildinfo written to python3-defaults_3.10.6-1~22.04_amd64.buildinfo
INFO: 	buildlog written to python3-defaults_3.10.6-1~22.04_arm64.buildlog
INFO: 	Hash of python3-defaults_3.10.6-1~22.04_amd64.buildinfo matches hash in changes file
```

**Comment by philroche** [comment]
ack. I can replicate.

**Comment by philroche** [comment]
0.0.5 now released https://pypi.org/project/ubuntu-package-buildinfo/0.0.5/ to resolve this issue with added support for builds that are `Architecture: all` which will always be built as part of an adm64 build.

**Comment by djcampello** [comment]
Closing this issue as it seems to have been resolved with implementation on 0.0.5. Thank you so much!
